tktermwidget/tkterm.py

```diff
@@ -93,70 +107,94 @@ def __init__(self, master: Misc, autohide: bool = True, *args, **kwargs):
 
         # Grid widgets
         self.text.grid(row=0, column=0, sticky="nsew")
-        self.xscroll.grid(row=1, column=0, sticky="ew")
+        if kwargs.get("wrap", "char") == "none":
+            self.xscroll.grid(row=1, column=0, sticky="ew")
         self.yscroll.grid(row=0, column=1, sticky="ns")
 
         # Create command prompt
         self.directory()
 
         # Set variables
-        self.index = 1
+        self.longsymbol = "\\" if not SYSTEM == "Windows" else "\\"
+        self.index, self.cursor = 1, self.text.index("insert")
```

Comment:
Separate and add type annotations to all the variables.

Reply:
fixed